### PR TITLE
Fix template links in azure.html

### PR DESCRIPTION
**Pull request by lingmann**
Update the azure.html page to render links using the configured Azure
release location (dcosio.azureedge.net). FIXES: https://mesosphere.atlassian.net/browse/DCOS-9357

### CODE REVIEW

**Comment by lingmann** [comment]
@cmaloney this is ready for review. The current `deploy-vpc-api` test failure looks to be flaky/unrelated...

File: gen/installer/azure.py [hunk 1]
```diff
@@ -20,7 +22,7 @@
 
 TEMPLATE_PATTERN = re.compile('(?P<pre>.*?)\[\[\[(?P<inject>.*?)\]\]\]')
 
-UPLOAD_URL = ("https://az837203.vo.msecnd.net/dcos/{channel_commit_path}/azure/{arm_template_name}")
```
Comment (lingmann) [inline gen/installer/azure.py]: This is the main fix... we still had some remnants of the Azure storage location hard-coded.

Comment (cmaloney) [inline gen/installer/azure.py]: Heh, I was supposed to fix that long ago :P. Thanks for taking care of it now

**Comment by cmaloney** [comment]
Have you manually tested launching with https://downloads.dcos.io/dcos/testing/pull/634/azure.html?

**Comment by lingmann** [comment]
@cmaloney yup works great

**Comment by cmaloney** [comment]
:ship:

**Pull request merged**